src/pages/Home.jsx

```diff
@@ -42,7 +42,7 @@ const Home = () => {
             <HomeSection
               isReallySmallScreen={isReallySmallScreen}
               src="/assets/web-security-green.png"
-              alt="Web security"
+              alt=""
```

Comment:
It seems like this is a prop used to pass in the description for the image alt encased in the component. Think we could rename this to componentImgAlt or something. Same with src.

Reply:
Yeah good point. I'll get that.